### PR TITLE
Homepage Redesign - Position Sections

### PR DESCRIPTION
**Pull request by mjoyce91**
Adds the New and Popular Positions sections to the Homepage.

What this PR covers:
- Presentation layer of new and popular positions on the home page
- Appropriate prop types
- "View More" link

What this PR does not cover:
- Tests
- Passing real data as props to the components (i.e. the actual results from the API)
- Any real action associated with the "Add to Bid List" button
- How to check if a position is considered "new"
- Actual results property names for placeholders
- A URL to use for Popular Positions "View More" link (straight `/results` for now)

### CODE REVIEW

File: src/Components/PositionsSectionContent/PositionsSectionContent.jsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
```
Comment (burgwyn) [inline src/Components/PositionsSectionContent/PositionsSectionContent.jsx]: This file is uncovered

File: src/Components/PositionsSectionContent/index.js [hunk 2]
```diff
@@ -0,0 +1 @@
+export { default } from './PositionsSectionContent';
```
Comment (burgwyn) [inline src/Components/PositionsSectionContent/index.js]: Let's add `!src/Components/**/index.js` to our `package.json > jest > collectCoverageFrom` so we don't worry about these files in our coverage

File: src/Components/PositionsSectionContent/PositionsSectionContent.jsx (hunk 1 above)
Comment (mjoyce91) [reply src/Components/PositionsSectionContent/PositionsSectionContent.jsx]: I decided to not do any tests to split things up evenly between two PRs.

File: src/Components/PositionsSectionContent/index.js (hunk 2 above)
Comment (mjoyce91) [reply src/Components/PositionsSectionContent/index.js]: Good point, I forgot to mention that I introduced this style of importing components. Makes the references shorter (`'../PositionsSectionContent/PositionsSectionContent'` -> `'../PositionsSectionContent'`).

Comment (mjoyce91) [reply src/Components/PositionsSectionContent/index.js]: Updated

**Comment by burgwyn** [comment]
Can you commit `yarn.lock`?

File: package.json [hunk 3]
```diff
@@ -22,6 +22,7 @@
     "redux-saga": "^0.15.4",
     "redux-thunk": "^2.2.0",
     "shortid": "^2.2.8",
+    "typeface-roboto": "0.0.35",
```
Comment (burgwyn) [inline package.json]: Need the updated `yarn.lock`

**Pull request merged**